docs/topics/wasm/wasm-debugging.md

```diff
@@ -133,41 +133,42 @@ kotlin {
 >
 {style="note"}
 
-#### Enhance your debugging with custom formatters
+#### Use custom formatters
 
-In addition to the default debugging configuration, you can set up custom formatters to display and locate variable values in a more user-friendly and comprehensible manner.
+Custom formatters help display and locate variable values in a more user-friendly and comprehensible manner when debugging Kotlin/Wasm code.
 
-![Kotlin/Wasm improved debugger](wasm-debugger-improved.png){width=600}
+Custom formatters are enabled by default in development builds, so you don't need additional Gradle configurations. 
 
-This implementation is supported across major browsers like Firefox and Chromium-based as
+This feature is supported across major browsers like Firefox and Chromium-based as
 it uses the [custom formatters API](https://firefox-source-docs.mozilla.org/devtools-user/custom_formatters/index.html).
 
-To set up custom formatters for an improved debugging experience:
+To use this feature, you only need to ensure that custom formatters are enabled in your browser's developer tools:
 
-1. Add the following compiler argument to the `wasmJs` compiler options:
+* In the Chrome DevTools, it's placed in **Settings | Preferences | Console**:
```

Comment:
```suggestion
* In Chrome DevTools, it's placed in **Settings | Preferences | Console**:
```

In their docs, "Chrome DevTools" is a product name which doesn't need an article.